doc/code/qml_qchem.rst

```diff
@@ -184,7 +185,7 @@ backend can be selected by setting ``method='pyscf'`` in :func:`~.molecular_hami
 
     symbols = ["H", "H"]
     geometry = np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 2.0]])
-    molecule = qml.qchem.Molecule(symbols, geometry, charge=0, mult=1, basis='sto-3g')
+    molecule = qml.qchem.Molecule(symbols, geometry, charge=0, mult=1, basis_name='sto-3g')
     hamiltonian, qubits = qml.qchem.molecular_hamiltonian(molecule, method='pyscf')
 
 The non-differentiable backend requires the ``OpenFermion-PySCF`` plugin to be installed by the user
```

Comment:
Is this still a requirement for both of the backends? Otherwise 
```suggestion
 The non-differentiable ``openfermion`` backend requires the ``OpenFermion-PySCF`` plugin to be installed by the user
```

Reply:
Modified the text.